src/segment-loader.js

```diff
@@ -433,7 +433,8 @@ export default class SegmentLoader extends videojs.EventTarget {
 
     // if the video has not yet played once, and we already have
     // one segment downloaded do nothing
-    if (!hasPlayed && bufferedTime >= 1) {
+
+    if (!hasPlayed && bufferedTime >= Config.GOAL_BUFFER_LENGTH_BEFORE_PLAYING) {
```

Comment:
We now have a function, `goalBufferLength` in MasterPlaylistController that may be a good spot for this to exist.